class.jetpack-debugger.php

```diff
@@ -18,6 +18,31 @@ private static function is_jetpack_support_open() {
 		}
 	}
 
+	private static function what_jetpack_plan() {
+		$plan = Jetpack::get_active_plan();
+		$plan = $plan['product_slug'];
+		switch ( $plan ){
+			case 'jetpack_free' :
+			default :
+				$plan = 'Free';
+				break;
+			case 'jetpack_premium' :
+			case 'jetpack_premium_monthly' :
+				$plan = 'Premium';
+				break;
+			case 'jetpack_personal' :
```

Comment:
Out of curiosity, is there a reason that we're ordering Personal after Premium?  Wouldn't it make more sense to order from lowest to highest?

Also, would it be at all useful for happiness to be able to easily tell if it's monthly/annual, and the expiration date?  Or do y'all just care what it is now?